client/lib/plans/constants.js

```diff
@@ -209,9 +209,9 @@ export const PLANS_LIST = {
 
 	[ PLAN_PREMIUM ]: {
 		getTitle: () => i18n.translate( 'Premium' ),
-		getAudience: () => 'Best for entrepreneurs', //PLANS A/B TEST: Translate if test passes
-		getBlogAudience: () => 'Best for professionals', //PLANS A/B TEST: Translate if test passes
-		getPortfolioAudience: () => 'Best for freelancers', //PLANS A/B TEST: Translate if test passes
+		getAudience: () => i18n.translate( 'Best for entrepreneurs' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Best for Entrepreneurs &amp; Freelancers:' )` *ES Score: 14*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/remove/plans-signup-test)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).